libs/features/mas/web-components/src/hydrate.js

```diff
@@ -222,6 +225,17 @@ export function processCTAs(fragment, merchCard, aemFragmentMapping, variant) {
     }
 }
 
+function processAnalytics(fragment, merchCard) {
+  const { tags } = fragment;
+  const cardAnalyticsId = tags?.find(tag => tag.startsWith(ANALYTICS_TAG))?.split('/').pop();
+    if(cardAnalyticsId) {
+      merchCard.setAttribute(ANALYTICS_SECTION_ATTR, cardAnalyticsId);
+      merchCard.querySelectorAll(`a[data-analytics-id]`).forEach((link) => {
+        link.setAttribute(ANALYTICS_LINK_ATTR, `${link.dataset.analyticsId}--${cardAnalyticsId}`);
```

Comment:
```suggestion
      merchCard.querySelectorAll(`a[data-analytics-id]`).forEach((link, index) => {
        link.setAttribute(ANALYTICS_LINK_ATTR, `${index}-${link.dataset.analyticsId}`);
```

Reply:
i think dubbing the card lh is not at mas responsibility scope. However it's nice for everyone to get link index within the card

Reply:
fixed